aggregator/baobab/peg.por.json

```diff
@@ -1,8 +1,8 @@
 {
-  "aggregatorHash": "0xdfc65be0daca54cfd5c081bb325d855e71b1a9cd6d2c5e44fcc1d7d0756f7811",
+  "aggregatorHash": "0x952f883b8d2fd47a790307cb569118a215ea45eb861cefd4ed3b83ae7550f8e8",
   "name": "PEG-POR",
   "address": "0x58798D6Ca40480DF2FAd1b69939C3D29d91b60d3",
-  "heartbeat": 3600000,
+  "heartbeat": 120000,
```

Comment:
@nick-bisonai Do we want to change the heartbeat value?

Reply:
I'm not changing the value, it's the heartbbeat currently used in baobab